### PR TITLE
TMDM-13202 FK : render in main tab in MetadataRepository

### PR DESCRIPTION
**Pull request by zfkang-talend**
Jira: https://jira.talendforge.org/browse/TMDM-13202

**What is the current behavior?** (You should also link to an open issue here)
ReferenceFieldMetadata have no property about isFKMainRender, which defined in studio "X_ForeignKey_NotSep"

**What is the new behavior?**
add property isFKMainRender in ReferenceFieldMetadata, the default value is false.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/annotation/XmlSchemaAnnotationProcessorState.java [hunk 1]
```diff
@@ -234,4 +236,12 @@ public String getDefaultValueRule() {
     public void setDefaultValueRule(String defaultValueRule) {
         this.defaultValueRule = defaultValueRule;
     }
+
+    public boolean isFKMainRender() {
+        return isFKMainRender;
+    }
+
+    public void setFKMainRender(boolean FKMainRender) {
```
Comment (lpw82gf) [inline main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/annotation/XmlSchemaAnnotationProcessorState.java]:
```
public void setFKMainRender(boolean isFKMainRender) {
        this.isFKMainRender = isFKMainRender;
    }
```

File: main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/ReferenceFieldMetadata.java [hunk 2]
```diff
@@ -110,6 +112,47 @@ public ReferenceFieldMetadata(ComplexTypeMetadata containingType,
                 Collections.<String>emptyList());
     }
 
+    public ReferenceFieldMetadata(ComplexTypeMetadata containingType,
```
Comment (VictorLi-talendbj) [inline main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/ReferenceFieldMetadata.java]: Do we have to add this new constructor? how about to change the existing one?

Comment (lpw82gf) [reply main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/ReferenceFieldMetadata.java]: Changing existing constructor may need more refactor, could we just using existing constructor and call a setter for the attribute?

Comment (lpw82gf) [reply main/plugins/org.talend.mdm.commmon/src/main/java/org/talend/mdm/commmon/metadata/ReferenceFieldMetadata.java]: It is acceptable for now, we could have the new constructor.

**Pull request merged**